src/main/java/stsjorbsmod/patches/RenderBurningBlockPatch.java

```diff
@@ -0,0 +1,40 @@
+package stsjorbsmod.patches;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.helpers.ImageMaster;
+import javassist.CannotCompileException;
+import javassist.expr.ExprEditor;
+import javassist.expr.MethodCall;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.powers.BurningPower;
+
+public class RenderBurningBlockPatch {
+    @SpirePatch(clz = AbstractCreature.class, method = "renderHealth")
+    public static class AbstractCreature_renderHealthPatch {
+        public static ExprEditor Instrument() {
```

Comment:
It feels like this would be much simpler as a prefix patch on `renderBlockIconAndValue`, no?

```java
@SpirePrefixPatch
public static void patch(AbstractCreature __this, SpriteBatch sb, float x, float y) {
    if (__this.hasPower(BurningPower.POWER_ID)) {
        sb.setColor(Color.GOLD.cpy());
        sb.draw(
            ImageMaster.ATK_FIRE,
            x + AbstractCreature.BLOCK_ICON_X - 32.0F,
            y + AbstractCreature.BLOCK_ICON_Y - 32.0F + __this.blockOffset,
            32.0F,
            32.0F,
            64.0F,
            64.0F,
            Settings.scale,
            Settings.scale,
            0.0F);
    }
}
```